lib/benchee/benchmark/runner.ex

```diff
@@ -1,12 +1,12 @@
 defmodule Benchee.Benchmark.Runner do
-  @moduledoc """
-  This module actually runs our benchmark scenarios, adding information about
-  run time and memory usage to each scenario.
-  """
+  @moduledoc false
```

Comment:
going towards 1.0 I think we should also be more vigilant what we want to be official API vs our internal stuff.

Also mainly did this because I made some methods here public to be used from our repeating code.

Reply:
👍